0011_saslauthd_ac_prog_libtool.dpatch by  <fabbe@debian.org>

Enable libtool use.

diff -urNad trunk~/saslauthd/configure.in trunk/saslauthd/configure.in
--- trunk~/saslauthd/configure.in	2006-05-29 22:52:42.000000000 +0300
+++ trunk/saslauthd/configure.in	2006-11-01 23:41:51.000000000 +0200
@@ -25,6 +25,7 @@
 AC_PROG_MAKE_SET
 AC_PROG_LN_S
 AC_PROG_INSTALL
+AC_PROG_LIBTOOL
 
 dnl Checks for build foo
 CMU_C___ATTRIBUTE__
